src/components/molecules/cheatSheets/TagManagementCheatSheet.tsx

```diff
@@ -0,0 +1,23 @@
+import { useTranslation } from 'react-i18next'
```

Comment:
**[non-blocker]**

For future reference, there is no need to add these for now, they will be stored in a different structure, but no need to remove this